README.md

```diff
@@ -13,46 +13,111 @@ The image does not contain an entry point, because the build runner executes the
 
 # Azure Pipelines example
 
-The following build script builds the hello world ESP32 firmware inside the *infinitecoding/platformio-for-ci* Docker container.
+The following build script builds the project ESP32 firmware in the [example](example) folder withing the *infinitecoding/platformio-for-ci* Docker container with azure pipelines.

```

Comment:
I would remove the word project. As of now this does not make sense for me :)

Reply:
Line 59 the same

Reply:
done